lib/net/hostlist.js

```diff
@@ -1635,7 +1636,7 @@ function concat32(left, right) {
 }
 
 function random(max) {
-  return Math.floor(Math.random() * max);
+  return bcrypto.random.randomRange(0, max + 1);
 }
```

Comment:
Since `bcrypto.random.randomRange` is inclusive with its first argument and exclusive with its second argument, need to do `max + 1`

Reply:
This isn't true based on how the `random` function is used in practice. The length of an array is passed in and then the returned result is used to index into the array. This means that the argument isn't actually `max` since it should never return the "max" value passed in.